### ISSUE TITLE
Add AtlasPack atlases to fs-tabulate

### ISSUE THREAD

**Issue by tsalo** [PennLINC/freesurfer_tabulate#8]
We probably want parcellated results using the AtlasPack 4S atlases as well, right?

**Comment by mattcieslak** [comment]
Good point, although fs-tabulate is only gathering surface label summaries, so the 2-4th Ss won't be included here

**Comment by tsalo** [comment]
What about subcortical regions' volumes and things like that?

**Comment by mattcieslak** [comment]
I think freesurfer only does volume stats on regions that they "aseg". To get the size of each subcortical region from the 4S atlas we'd need a nonlinear warp from a compatible MNI, which freesurfer won't have. But fmriprep/qsiprep will.

We may want to add a separate stats gatherer for these that can be optionally run.

**Comment by tsalo** [comment]
Would sMRIPrep have the warp? If so, refactoring this into an sMRIPost workflow (i.e., one that ingests sMRIPrep derivatives) could solve that problem.

EDIT: Or, I guess there could be a separation of concerns here, with one tabulation App and one sMRIPost App.

**Comment by mattcieslak** [comment]
smripost is probably the best spot for calculating the volume of those regions, because it will have all the needed files on hand for sure

**Comment by tsalo** [comment]
I thought we were going to make fs-tabulate into an sMRIPost App? Would you prefer to have two Apps instead?

**Comment by mattcieslak** [comment]
This issue is closed by freesurfer-post, which has instructions on linking the 4S atlases to the original schaefer parcellations
